src/g3log/loglevels.hpp

```diff
@@ -94,11 +91,7 @@ namespace g3 {
    static const int kInternalFatalValue = 2000;
 } // g3
 
-#if __cplusplus >= 202002L
-   constexpr
-#else
-   const
-#endif
+const
```

Comment:
please make this the same line as `LEVELS G3LOG_DEBUG{g3::kDebugValue, "DEBUG"},`

Ref how it was earlier https://github.com/KjellKod/g3log/commit/cc0fb7c1ea7d7cc2bc5d8766b18b190ba74b70f9